cplusplus.json

```diff
@@ -813,5 +813,145 @@
+  {"id": "cpp.better-enums", "name": "Better Enums", "tech": ["utils"], "tags": ["enums"], "repo": "aantron/better-enums",
+  "imports": ["enum.h"], "examples": ["https://github.com/aantron/better-enums/tree/master/example"]},
+  {"id": "cpp.bstring", "name": "Better String Library", "tech": ["utils"], "tags": ["strings"], "repo": "msteinert/bstring",
+  "imports": ["bst"], "examples": ["https://github.com/msteinert/bstring/tree/master/tests"]},
+  {"id": "cpp.boost-signals2", "name": "Boost.Signals2", "tech": ["app-arch"], "tags": ["signals-slots"], "repo": "boostorg/signals2",
+  "imports": ["boost/signals2/"], "examples": ["https://github.com/boostorg/signals2/tree/develop/example"]},
+  {"id": "cpp.fmt", "name": "fmt", "tech": ["eng-tools"], "tags": ["formatter"], "repo": "fmtlib/fmt",
+  "imports": ["fmt/"], "examples": ["https://github.com/fmtlib/fmt/tree/master/test"]},
+  {"id": "cpp.fastformat", "name": "FastFormat", "tech": ["eng-tools"], "tags": ["formatter"], "repo": "https://sourceforge.net/projects/fastformat/files/fastformat/0.7/0.7.1/0.7.1/fastformat-0.7.1.zip/download",
+  "imports": ["fastformat/"], "examples": ["https://sourceforge.net/projects/fastformat/files/fastformat/0.7/0.7.1/0.7.1/fastformat-0.7.1.zip/download"]},
+  {"id": "cpp.casacore", "name": "casacore", "tech": ["data-science"], "tags": ["astronomy"], "repo": "casacore/casacore",
+  "imports": ["aips.h", "Coordinates.h", "DerivedMC.h", "FITS.h", "Images.h", "Lattices.h", "MeasUDF.h", "Measures.h", "MS", "MSFits.h", "Converters.h", "Mathematics.h", "Tables.h"],
+  "examples": ["https://github.com/casacore/casacore/tree/master/msfits/apps"]},
+  {"id": "cpp.qt-verbal-expressions", "name": "QtVerbalExpressions", "tech": ["utils"], "tags": ["regex"], "repo": "VerbalExpressions/QtVerbalExpressions",
+  "imports": ["QVerbalExpressions.h"], "examples": ["https://github.com/VerbalExpressions/QtVerbalExpressions/tree/master/example"]},
+  {"id": "cpp.libcsptr", "name": "C Smart Pointers", "tech": ["faster-code"], "tags": ["smart-pointer"], "repo": "Snaipe/libcsptr",
+  "imports": ["csptr/"], "examples": ["https://github.com/Snaipe/libcsptr/tree/master/check/test"]},
+  {"id": "cpp.cxx-prettyprint", "name": "cxx-prettyprint", "tech": ["eng-tools"], "tags": ["pretty-output"], "repo": "louisdx/cxx-prettyprint",
+  "imports": ["prettyprint.hpp"], "examples": ["https://github.com/louisdx/cxx-prettyprint/blob/master/ppdemo.cpp"]},
+  {"id": "cpp.gcc-poison", "name": "gcc-poison", "tech": ["reliable-code"], "tags": ["unsafe-functions"], "repo": "leafsr/gcc-poison",
+  "imports": ["poison.h"], "examples": ["https://github.com/leafsr/gcc-poison#example-usage"]},
+  {"id": "cpp.kangaru", "name": "kangaru", "tech": ["app-arch"], "tags": ["inversion-of-control"], "repo": "gracicot/kangaru",
+  "imports": ["kangaru/kangaru.hpp"], "examples": ["https://github.com/gracicot/kangaru/tree/master/examples"]},
```

Comment:
maybe just kangaru/ ? seems specific enough

Reply:
you mean, one can include individual headers if needed? anyway no problem.